src/main/kotlin/blackjack/Card.kt

```diff
@@ -0,0 +1,19 @@
+package blackjack
+
+class Card(val suit: CardSuit, val number: Int) {
+    fun numberToCardDeckElements(number: Int): String {
+        return when (number) {
+            1 -> "A"
+            10 -> "J"
+            11 -> "Q"
+            12 -> "K"
+            else -> number.toString()
+        }
+    }
+
+    override fun toString(): String {
+        val cardValue = numberToCardDeckElements(number)
+        return "%s%s".format(cardValue, suit.value)
+    }
```

Comment:
The domain object is currently handling code that's only needed in the View layer. Try to decouple the domain from the View by moving that logic into the View layer instead. 🙂